sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c

```diff
@@ -34,6 +35,15 @@
  * the address of the access.
  */
 
+/**
+ * Secure MMIO context.
+ *
+ * This is placed at a fixed location in memory within the .static_critical
+ * section. The location of this data is known to ROM_EXT.
+ */
+__attribute__((section(".static_critical.sec_mmio_ctx"), used))  //
```

Comment:
What does `used` get us here? ISTR that `used` kinda doesn't work unless the object it's in is an input object (not a library) tio the linker?

Reply:
Good catch, yeah they aren't needed.